CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # CHANGELOG
 
 # Version 2.4.0 (Unreleased)
-- Fixed [#749](https://github.com/Microsoft/ApplicationInsights-Java/issues/749) introduce support for postgresql jdbc4 prepared statements.
+- Deprecated support for multiple apps instrumented with single JVM Agent. Instrumentation will only work for single apps
```

Comment:
Change "Deprecated" to "Removed".

I would also remove the second sentence. I think what will happen is the first application to register itself with the agent will be the one sending the telemetry collected from the agent; e.g. **app1** and **app2** register with the agent, **app1** registers first, all telemetry from dependency instrumentation in **app2** will appear to be sent from **app1**.

I'm concerned this may be a change for a major version.

Reply:
This is correct. This is what will happen. 
Re change for version: we plan to release beta wit this change. If there would be more votes in community to bring back the old behavior we will work on that. Currently, the multi-app scenario is already broken and doesn't work in multi-threaded environment. Modern apps are multi-threaded and we need to support those needes.

Reply:
Cool. Let's definitely release this as beta first. If someone is deploying multiple applications to a single application server, I want to know ahead of time.

Since we are doing that, I'm going to cherry-pick out changes unrelated to this PR and merge them separately. If we do end up having to revert, it'll be nicer to have one commit to revert.

Reply:
Sounds great! Thanks @littleaj